wsagent/che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConnection.java

```diff
@@ -1622,14 +1627,19 @@ protected JSch getJSch(final OpenSshConfig.Host hc, FS fs) throws JSchException
                     String password = remoteUrl.substring(remoteUrl.lastIndexOf(":") + 1, remoteUrl.indexOf("@"));
                     command.setCredentialsProvider(new UsernamePasswordCredentialsProvider(username, password));
                 } else {
+                    String gitUser = request.getAttributes().get("userName");
```

Comment:
yes, but in other places username is one word https://github.com/eclipse/che/blob/jpa-integration/wsmaster/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/UserValidator.java#L137-137
https://github.com/eclipse/che/blob/jpa-integration/wsmaster/che-core-api-auth/src/main/java/org/eclipse/che/security/oauth/OAuthAuthenticationService.java#L87-87